src/components/FormWrapper/FormWrapper.tsx

```diff
@@ -0,0 +1,47 @@
+import { FormEvent } from 'react';
+import { Link } from 'react-router-dom';
+import clsx from 'clsx';
+
+import { ChildrenFC } from '../../utils/type';
+import style from './FormWrapper.module.scss';
+
+type FormWrapperProps = {
+  children: React.ReactNode;
+  title: string;
+  onSubmit: (event: FormEvent<HTMLFormElement>) => void;
+  formRef?: React.RefObject<HTMLFormElement>;
+  className?: string;
+  redirectText?: string;
+  redirectTo?: string;
+  redirectLinkText?: string;
+  illustration?: string;
+  variant?: 'default' | 'register';
+};
+
+export const FormWrapper: ChildrenFC<FormWrapperProps> = ({
+  children,
+  title,
+  onSubmit,
+  formRef,
```

Comment:
Perfectly fine to name it this way. If it would be a Button or Input, or some "leaf" component. I would suggest going for `ref` even for the cost of the RADICULOS (British accent, definitely spelled this way) `forwardRef`.

Creating the most used component with API that obeys the `principle of least surprise` is paramount.